styles/contact.css

```diff
@@ -11,24 +11,34 @@
     --white: #fff;
 }
 
+.contact-container{
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
 
-.contact-container {
+}
+/* .contact-container {
```

Comment:
commented out code - do we still need this? should we label what it is if we are going to keep it commented out and here?